src/analysis/seminorm.lean

```diff
@@ -534,8 +534,8 @@ nonneg_of_mul_nonneg_left h zero_lt_two
 lemma sub_rev : p (x - y) = p (y - x) := by rw [←neg_sub, p.neg]
 
 lemma le_insert : p y ≤ p x + p (x - y) :=
-calc p y = p (x - (x - y)) : by abel
-... ≤ p x + p (x - y) : by exact p.sub_le _ _
+calc p y = p (x - (x - y)) : by rw sub_sub_cancel
+... ≤ p x + p (x - y) : p.sub_le _ _
```

Comment:
```suggestion
eq.trans_le (congr_arg p (sub_sub_cancel _ _).symm) (p.sub_le _ _)
```
I really like term-mode, but whether this is better is not really clear.

Reply:
Go for it fully then!
```suggestion
(congr_arg p (sub_sub_cancel _ _).symm).trans_le $ p.sub_le _ _
```
but yeah it's not an obvious improvement.

Reply:
Well, the proof of norm_le_insert was written in calc mode so that's why I kept it this way

Reply:
I really don't see the point of obfuscating this proof. I would keep Pierre-Alexandre's proof.